SU2_PY/SU2/run/merge.py

```diff
@@ -62,7 +62,7 @@ def merge( config ):
     
     # check if needed
     partitions = konfig['NUMBER_PART']
-    if partitions <= 1:
+    if partitions > 1:
```

Comment:
Maybe I am confused here, but this change contradicts the Assumptions listed above.